README.md

```diff
@@ -1,56 +1,93 @@
 # tdom - WIP
 
-PEP750 based t strings for both SSR and FE
+PEP 750 based t strings for both SSR and FE.
 
 [Live demo](https://webreflection.github.io/tdom/src/)
 
-## Python setup
+## Installation
 
-- Clone the main branch then `cd tdom`
-- Make a .venv (or just `. env.sh` once)
-- `uv sync` (or just `. env.sh` once)
+We don't yet have a package published on PyPI, so follow the instructions below. Once template strings are merged into
+a Python 3.14 alpha, we'll publish a package, and you can install from `pip`.
 
-### Custom Python
+## Python 3.14 setup
 
-To test a custom Python build link the executable into the `./env/bin` folder, replacing the `python` reference in there.
+*Note: These instructions point at an old version of template strings. We need some Pyodide work to start using the new
+API. So we'll use the old API for the CPython by cloning an old commit.*
 
+First, clone Lysandros's CPython fork with the older implementation of t-strings, then build it in some directory.
 
+```shell
+$ cd /tmp
+$ https://github.com/lysnikolaou/cpython.git
+$ git checkout b07c4191720e318e25fcba2be93d6b7b0aedc668
+$ ./configure
+$ ./make

```

Comment:
this is just `make` ???

Reply:
It works fine for me, did it not work for you?

Reply:
I'm an idiot. Of _course_ it's not `./make`. Yes, I'll fix when getting rebase done.